CONTRIBUTING.md

```diff
@@ -318,6 +318,21 @@ Python dependencies in Kedro-Viz are usually updated automatically through tools
 
 If the lower-bound e2e test fails, it indicates that some dependencies may not work correctly with Kedro-Viz. To resolve this, update the problematic dependency in both `requirements.txt` and `lower-requirements.txt`.
 
+### Testing UI using Gitpod
```

Comment:
I think we should add a step to 'Install Gitpod' from chrome store

Reply:
ah yea good idea. I included it now